CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 Changes by Version
 ==================
 
+0.43.0
+-------------------
+* Bumped OpenTelemetry Collector to v0.43.0
+* Upgrade to 0.43.0 will move the metrics CLI arguments into the config, in response to ([#4695](https://github.com/open-telemetry/opentelemetry-collector/pull/4695), [@codebote](https://github.com/codeboten))
```

Comment:
this is from the collector not operator repo

Reply:
I've changed to the release PR.